### ISSUE TITLE
Dependency property

### ISSUE THREAD

**Issue by pumba-lt** [ewilderj/doap#28]
DOAP does not seem to include any properties to describe relationships (dependencies) between projects and/or their releases.

**Comment by sshikov** [comment]
I think, dependencies support should be at least as capable as in maven pom.xml - just implemented in RDF way.

**Comment by ewilderj** [comment]
Yes, I left dependencies out in the first case of writing DOAP, because I wanted to solve the simpler problem first. Dependencies somewhat relies on the underlying packaging system, so I'm not yet sure what the best way of integrating them actually is. 

It would be good if @pumba-lt and @sshikov you could describe some of your expected use cases.

**Comment by ewilderj** [comment]
OK, so getting myself back up to speed this is actually discussed in Issue #1 

TL;DR you can use http://ontologi.es/doap-deps# until such time as we merge dependency handling into DOAP.

Closing because further discussion should happen in #1

**Comment by ewilderj** [comment]
Duplicate of #1

**Comment by pumba-lt** [comment]
My use case was scraping Maven central repository to transform artifact metadata and their dependencies to RDF. I gave up because there was too much data.
